Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -245,8 +245,8 @@ def check_DESCRIPTION_min_length(fb, descfile):
 
 @register_condition
 @condition
-def metadata(font):
-  family_dir = os.path.dirname(font)
+def metadata(fonts):
+  family_dir = os.path.dirname(fonts[0])
```

Comment:
...perhaps we could unify all those with a condition called family_directory ?

Also, there's a check that verifies whether all fonts are in the same directory, so I consider it safe to use fonts[0] instead of any other element in the fonts list because all of them are expected to have the same base dir.

Another assumption here was that the fontbakery should simply halt with a critical error is the user passes an empty set of TTF files. I think it is better to check for that right in the beginning (in the testsuite runner script) instead of having to check for it again everywhere else. What do you think?